crates/viewer/re_view_spatial/src/proc_mesh.rs

```diff
@@ -513,8 +521,21 @@ impl Cache for SolidCache {
         self.0.clear();
     }
 
-    fn bytes_used(&self) -> u64 {
-        0 // mostly VRAM
+    fn memory_report(&self) -> CacheMemoryReport {
+        CacheMemoryReport {
+            bytes_cpu: self.0.total_size_bytes(),
+            bytes_gpu: Some(
+                self.0
+                    .values()
+                    .map(|mesh| mesh.as_ref().map_or(0, |m| m.gpu_mesh.gpu_byte_size()))
+                    .sum(),
+            ),
+            per_cache_item_info: Vec::new(),
+        }
+    }
+
+    fn name(&self) -> &'static str {
+        "Solid Cache"
```

Comment:
in context that name makes sense, but if I see this in the panel a week from now I'd be confused
Maybe `Proc Mesh Solids`?

On that note, applies to everywhere, having cache in the name is a bit redundant. On the other hand nice because it resolves better to the type name in the code. No strong opinion either way (your call, feel free to ignore that aspect 🤷)